File: jeff-farm-ui/src/app/crud/crud-item-map-update.ts
import { CrudItemCoordinate } from './crud-item-coordinate';
import { CrudItemMap } from './crud-item-map';

export class CrudItemMapUpdate {
    constructor(
        public group: CrudItemMap,
        public addCoordinates: CrudItemCoordinate[],
        public removeCoordinateIds: number[]) { }
}

File: jeff-farm-ws/src/main/java/com/github/ants280/jeff/farm/ws/model/CrudItemMapUpdate.java
package com.github.ants280.jeff.farm.ws.model;

public class CrudItemMapUpdate
{
	private CrudItemMap map;
	private CrudItemCoordinate[] addCoordinates;
	private int[] removeCoordinateIds;

	public CrudItemMap getMap()
	{
		return map;
	}

	public void setMap(CrudItemMap map)
	{
		this.map = map;
	}

	public CrudItemCoordinate[] getAddCoordinates()
	{
		return addCoordinates;
	}

	public void setAddCoordinates(CrudItemCoordinate[] addCoordinates)
	{
		this.addCoordinates = addCoordinates;
	}

	public int[] getRemoveCoordinateIds()
	{
		return removeCoordinateIds;
	}

	public void setRemoveCoordinateIds(int[] removeCoordinateIds)
	{
		this.removeCoordinateIds = removeCoordinateIds;
	}
}
